### PR TITLE
Reworks FeaturePerfCSCollection setup and includes synthetic projects

### PR DESCRIPTION
**Pull request by vulder**

### CODE REVIEW

File: src/MultiSharedMultipleRegions/CMakeLists.txt [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-  MSMRmain.cpp
-  FeatureHeader.cpp
-)
-
-set_target_properties(MultiSharedMultipleRegions PROPERTIES
-  CXX_STANDARD 17
-  CXX_STANDARD_REQUIRED ON
+set_target_properties(
+  MultiSharedMultipleRegions PROPERTIES CXX_STANDARD 17 CXX_STANDARD_REQUIRED
+                                                        ON
```
Comment (boehmseb) [inline src/MultiSharedMultipleRegions/CMakeLists.txt:7]: CMake-format, really? -.-

**Review by boehmseb — APPROVED** [review]
I assume the implementations of the projects follows later?

File: src/MultiSharedMultipleRegions/CMakeLists.txt (hunk 1 above)
Comment (vulder) [reply src/MultiSharedMultipleRegions/CMakeLists.txt:7]: Threshold not reached :cry:

**Review by LuAbelt — APPROVED** [review]

File: external/FeaturePerfUtilities
Comment (LuAbelt) [inline external/FeaturePerfUtilities:1]: Is it intended behavior that I can currently not access this repo?

Comment (vulder) [reply external/FeaturePerfUtilities:1]: No, this was not intended. Should be fixed now.

**Pull request merged**